h_transport_materials/property_database/tzm.py

```diff
@@ -0,0 +1,139 @@
+import h_transport_materials as htm
+from h_transport_materials import (
+    Permeability,
+    Diffusivity,
+    Solubility,
+    DissociationCoeff,
+    RecombinationCoeff,
+) 
+import numpy as np
+
+u = htm.ureg
+
+forcey_permeability_d = Permeability(
+    pre_exp=8.32e-8 * u.mol * u.m**-1 * u.Pa**-0.5 * u.s**-1,
+    act_energy=75.5 * u.kJ * u.mol**-1,
+    range=(515 * u.K, 742 * u.K),
+    isotope="D",
+    source="forcey_permeability_1993",
+    note="Equation 8",
+)
+
+tominetti_permeability_h = Permeability(
+    pre_exp=5.53e-5 * 6.90e-4 * u.mol * u.m**-1 * u.Pa**-0.5 * u.s**-1,
+    act_energy=(84.6 - 11.8) * u.kJ * u.mol**-1,
+    range=(673 * u.K, 873 * u.K),
+    isotope="H",
+    source="tominetti_solubility_1990",
+    note="D*Ks",
+)
+
+forcey_diffusivity_d = Diffusivity(
+    D_0=3.18e-4 * u.m**2 * u.s**-1,
+    E_D=88.1 * u.kJ * u.mol**-1,
+    range=(515 * u.K, 742 * u.K),
+    isotope="D",
+    source="forcey_permeability_1993",
+    note="Equation 9",
+)
+
+tominetti_diffusivity_d = Diffusivity(
+    D_0=1.96e-4 * u.m**2 * u.s**-1,
+    E_D=95.5 * u.kJ * u.mol**-1,
+    range=(673 * u.K, 873 * u.K),
+    isotope="D",
+    source="tominetti_solubility_1990",
+    note="Equation 9",
+)
+
+tominetti_diffusivity_h = Diffusivity(
+    D_0=5.53e-5 * u.m**2 * u.s**-1,
+    E_D=84.6 * u.kJ * u.mol**-1,
+    range=(673 * u.K, 873 * u.K),
+    isotope="H",
+    source="tominetti_solubility_1990",
+    note="Equation 8",
+)
+
+forcey_solubility_d = Solubility(
+    S_0=2.62e-4 * u.mol * u.m**-3 * u.Pa**-0.5,
+    E_S=12.6 * u.kJ * u.mol**-1,
```

Comment:
```suggestion
    E_S=-12.6 * u.kJ * u.mol**-1,
```

In Equation 10 the activation energy is negative (there is a + sign in the exponential)